manifest.webapp

```diff
@@ -61,6 +61,11 @@
       "description": "Required to select folders to synchronize on devices",
       "type": "io.cozy.files",
       "verbs": ["ALL"]
+    },
+    "contacts": {
```

Comment:
Why is this authorization needed?

Reply:
BC v64 : https://github.com/cozy/cozy-ui/releases/tag/v64.0.0

Reply:
As we don't use `ContactsListModal` nor `ContactPicker`, we shouldn't have to add this permission, should we?

Reply:
🤔 tu penses qu'il vaut mieux l'ajouter lors de l'utilisation, plutôt que "par défaut" ?

Reply:
Je dirais que oui, sinon toutes nos apps auraient besoin des permissions pour io.cozy.contacts non?
Après j'ai pas tout l'historique des choix faits sur les autorisations. Ca mérite peut être de poser la question sur le canal tech?

Reply:
> sinon toutes nos apps auraient besoin des permissions pour io.cozy.contacts non?

Oui c'est le problème en effet, effectivement j'imagine qu'on n'a pas le souci si on n'utilise pas le composant. J'avais rajouté l'info dans la doc https://docs.cozy.io/cozy-ui/react/#!/ContactsListModal et https://docs.cozy.io/cozy-ui/react/#!/ContactPicker, supprimons la permission ici.